app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
   before_action :confirm_two_factor_authenticated
   layout 'card_wide'
 
   def index
```

Comment:
ah you're right, for some reason I thought it was saying the controller would be singularly named, but I looked more closely. keeping plural controller!